.buildkite/pipeline.yml

```diff
@@ -0,0 +1,17 @@
+steps:
+  - label: "Swift Build and Tests"
```

Comment:
I wonder if `{{ matrix.xcode_version }}` could do in the `label` so that it's clear on the UI why there's two steps, and which one is which.

<img width="1166" alt="image" src="https://github.com/buildkite/test-collector-swift/assets/15759/67627426-8e6e-4083-910b-f86e5cfae6f9">

While you're there, a bit of emoji could be nice:
https://github.com/buildkite/emojis#emoji-reference
<img width="211" alt="image" src="https://github.com/buildkite/test-collector-swift/assets/15759/1a4bfc29-05cf-4ef9-a458-4edce2937768">

Maybe this?

```suggestion
  - label: ":swift: build & test (Xcode {{ matrix.xcode_version }})"
```

Reply:
Great suggestion! Very cool, PR updated

Reply:
This looks awesome 
![image](https://github.com/buildkite/test-collector-swift/assets/30752103/e2733a01-daa8-43fd-867a-384ac89ca6a4)

Reply:
<img width="1158" alt="image" src="https://github.com/buildkite/test-collector-swift/assets/15759/7389204e-81cb-48f0-9aaf-457b559541ff">

👌🏼